data/i18n/en/en.toml

```diff
@@ -279,6 +279,9 @@ other = "Thanks for the feedback. If you have a specific, answerable question ab
 [layouts_docs_search_fetching]
 other = "Fetching results..."
 
+[legacy_repos_message]
+other = """Kubernetes has [new package repositories hosted at `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/) starting from August 2023. The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) have been deprecated and frozen starting from September 13, 2023. To install Kubernetes versions released **after** September 13, 2013, you need to use the community-owned package repositories. Please read our [deprecation and freezing announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details. The new Kubernetes package repositories provide downloads for Kubernetes versions starting with v1.24.0."""
```

Comment:
nit: avoid using “we” (and therefore “our”)

Reply:
Replaced `our` with `the`, PTAL again